bots/engine.py

```diff
@@ -196,6 +198,7 @@ def start():
 
         cleanup.cleanup(do_cleanup_parameter,userscript,scriptname)
     except Exception as msg:
+        print('Severe error:' + msg)
```

Comment:
not needed, is via logger

Reply:
Added these for quicker debugging while testing, it's good to see errors on the console